amethyst_config/examples/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "display": {
+    "title": "Amethyst game",
+    "brightness": 1.0,
+    "fullscreen": false,
+    "dimensions": [
+      1024,
+      768
+    ],
+    "min_dimensions": null,
+    "max_dimensions": null,
+    "vsync": true,
+    "multisampling": 0,
+    "visibility": true
+  },
+  "logging": {
+    "file_path": "new_project.log",
+    "output_level": "warn",
+    "logging_level": "debug"
+  }
+}
```

Comment:
Newline

Reply:
I'm not sure what you mean? This is the output from serde json

Reply:
It's usual practice for files to end with a newline as most of the diff tools expect this format. Github also reminds you of this if you look at the changes tab.

But if it's how serde outputs then leave it as is